src/Proj4.jl

```diff
@@ -34,7 +34,7 @@ end
 
 "Get human readable error string from proj.4 error code"
 function _strerrno(code::Cint)
-    bytestring(ccall((:pj_strerrno, libproj), Cstring, (Cint,), code))
+    bytestring(ccall((:pj_strerrno, libproj), Ptr{Cchar}, (Cint,), code))
```

Comment:
I think that's because it's not defined via the `@compat` macro - it's just a constant type alias.  See
https://github.com/JuliaLang/Compat.jl/blob/bc9f16c6e582345df8517d4f0708fff8138c2baa/src/Compat.jl#L529